src/renderer/actions/validator.ts

```diff
@@ -1,24 +1,59 @@
-import {ValidatorResponse} from "@chainsafe/lodestar-types";
+import {Gwei} from "@chainsafe/lodestar-types";
 import {Action, Dispatch} from "redux";
+
 import {ValidatorActionTypes} from "../constants/action-types";
+import {IValidator} from "../containers/Dashboard/DashboardContainer";
 import {IRootState} from "../reducers";
 import {fromHex} from "../services/utils/bytes";
 
-export const loadValidator = (validator: string) => {
+export interface ILoadValidators {
+    type: typeof ValidatorActionTypes.LOAD_VALIDATORS,
+    payload: Array<IValidator>,
+}
+
+export const loadValidatorsAction = () => {
+    return async (dispatch: Dispatch<Action<unknown>>, getState: () => IRootState): Promise<void> => {
+        const auth = getState().auth;
+        if (auth && auth.account) {
+            const validators = auth.account.getValidators();
+            const validatorArray = validators.map((v) => ({
+                name: auth.account!.name,
+                status: "TODO status",
+                publicKey: v.publicKey.toHexString(),
+                deposit: 30,
+                network: auth.account!.getValidatorNetwork(v.publicKey.toHexString()),
+                privateKey: v.privateKey.toHexString()
+            }));
+
+            dispatch({
+                type: ValidatorActionTypes.LOAD_VALIDATORS,
+                payload: validatorArray,
+            });
+        }
+    };
+};
+
+export const loadValidatorBalance = (validator: string) => {
```

Comment:
Might be better if we fetch balance of multiple validators, which allows us to use single api request to fetch them all later on